pkgs/development/perl-modules/generic/default.nix

```diff
@@ -5,6 +5,7 @@
 , outputs ? [ "out" "devdoc" ]
 , src ? null
 
+, enableParallelBuilding ? true
```

Comment:
Does this enable parallel building for perl pkgs by default?

Reply:
it does nothing.

i changed it to false and added a comment